students/johnpharmd/lesson04/trigrams.py

```diff
@@ -28,30 +28,86 @@
 
 # Data
 sample_st = 'I wish I may I wish I might'
+read_data = []
 s1 = ''
 s2 = ''
+st_key = ''
 s3 = ''
-count = 0
-st_set = set()
 st_dict = {}
+count = 0
 pron_set = set(['I', 'you', 'she', 'he', 'it', 'we', 'they'])
-
+n = 0
+new_st_lst= []
+testfile_dict = {1: 'sherlock_small.txt', 2: 'sherlock.txt'}
```

Comment:
normally the top of the file would only include constants, or instantiating logging module for example to make logging accessible in the whole module.

most of these should live with the function where they are used, consider readability aspect - every time you would need to go search for definition of vars instead of them being right where they are used.